packages/react-on-rails-pro/src/cache/TieredCacheHandler.ts

```
@@ -0,0 +1,82 @@
+
+    const l1Entry = this.applyL1Ttl(entry);
+    await Promise.all([l2Write, this.l1.set(key, l1Entry)]);
+  }
+
+  private applyL1Ttl(entry: CacheEntry): CacheEntry {
+    if (this.l1MaxTtlSeconds === undefined) return entry;
+
+    const capped =
+      entry.revalidate > 0
+        ? Math.min(entry.revalidate, this.l1MaxTtlSeconds)
+        : this.l1MaxTtlSeconds;
```

ESLint

[error] Replace `⏎········?·Math.min(entry.revalidate,·this.l1MaxTtlSeconds)⏎·······` with `·?·Math.min(entry.revalidate,·this.l1MaxTtlSeconds)`

(prettier/prettier)